src/Lean/Parser/Term.lean

```diff
@@ -555,11 +555,18 @@ def matchAltsWhereDecls := leading_parser
 /-- Similar to `binrel`, but coerce `Prop` arguments into `Bool`. -/
 @[builtin_term_parser] def binrel_no_prop := leading_parser
   "binrel_no_prop% " >> ident >> ppSpace >> termParser maxPrec >> ppSpace >> termParser maxPrec
-@[builtin_term_parser] def binop  := leading_parser
+@[builtin_term_parser] def binop := leading_parser
   "binop% " >> ident >> ppSpace >> termParser maxPrec >> ppSpace >> termParser maxPrec
-@[builtin_term_parser] def binop_lazy  := leading_parser
+-- like `binop%` but wrap arguments in `fun () => _`
```

Comment:
I think this should be a docstring, so that it appears when you hover over `binop_lazy%` in the source.

Reply:
It's not a good docstring though :) . We should add self-contained docstrings to all of these but that's a task for another PR

Reply:
I've added docstrings in #2778